docs/entities/TrialSession.md

```diff
@@ -173,19 +359,160 @@
       type: "string"
       flags: 
         presence: "required"
+      rules: 
+        - 
+          name: "min"
+          args: 
+            limit: 1
+        - 
+          name: "max"
+          args: 
+            limit: 4
     trialClerk: 
       type: "object"
       flags: 
         presence: "optional"
+      keys: 
+        name: 
+          type: "string"
+          flags: 
+            presence: "required"
+          rules: 
+            - 
+              name: "min"
+              args: 
+                limit: 1
+            - 
+              name: "max"
+              args: 
+                limit: 100
+        userId: 
+          type: "string"
+          flags: 
+            presence: "required"
+          rules: 
+            - 
+              name: "min"
+              args: 
+                limit: 1
+            - 
+              name: "guid"
+              args: 
+                options: 
+                  version: 
+                    - "uuidv4"
     trialLocation: 
-      type: "string"
+      type: "alternatives"
       flags: 
         presence: "required"
+      matches: 
+        - 
+          schema: 
+            type: "string"
+            flags: 
+              only: true
+            rules: 
+              - 
+                name: "min"
+                args: 
+                  limit: 1
+            allow: 
+              - "Fresno, California"
+              - "Tallahassee, Florida"
+              - "Pocatello, Idaho"
+              - "Peoria, Illinois"
+              - "Wichita, Kansas"
+              - "Shreveport, Louisiana"
+              - "Portland, Maine"
+              - "Billings, Montana"
+              - "Albany, New York"
+              - "Syracuse, New York"
+              - "Bismarck, North Dakota"
+              - "Aberdeen, South Dakota"
+              - "Burlington, Vermont"
+              - "Roanoke, Virginia"
+              - "Cheyenne, Wyoming"
+              - "Birmingham, Alabama"
```

Comment:
@ericsorenson 
It looks like this was arranged as alphabetical by state, but line 420-434 are separate from the larger ordered list. Any reason you can't keep lines 420-434 alphabetically ordered within the rest of the larger list?

Reply:
This is just an auto-generated file used for documentation. The new additions are only used for legacy case support.